test-configs.yaml

```diff
@@ -58,6 +58,8 @@ test_plan_default_filters:
       keys: ['arch', 'defconfig']
       values:
         - ['arm', 'multi_v7_defconfig']
+        - ['arm', 'versatile_defconfig']
```

Comment:
I don't think we should be enabling these defconfigs by default for all test plans.  If you want `boot_qemu` to be run with them, then add this filter to `boot_qemu`.  If it's for all QEMU devices, then it may be better to update the filters in the device types.